@shared/api/internal.ts

```diff
@@ -280,14 +279,32 @@ export const getAccountBalances = async ({
     networkPassphrase,
   });
 
-  let balances = null;
+  let balances: any = null;
```

Comment:
I played around with adding another type to BalanceMaps that would account for the LiquidityPool data. Something like
```
export interface BalanceMap {
  [key: string]: AssetBalance | NativeBalance | LPBalance
...
``` 
but the breaking change gets pretty ugly since LP data wouldn't share a lot of fields with AssetBalance and NativeBalance (eg. token), so it would need to be typecasted every time it's used. 

I think we'll need to take a holistic view of the wallet-sdk balance types and see how we can best incorporate a new LP share balance type (alternatively we could just rely on Horizon's BalanceLine type). But for now we're just displaying the shares so I think it's ok to set that aside as tech debt. Happy to discuss though

Reply:
Ah yeah, I was having some related issue with my these types with my work. Totally fine with using `any`. You'll see in my PR that doing it the "right way" is kinda ugly